test_requirements/base.txt

```diff
@@ -1,7 +1,23 @@
 aldryn-apphook-reload>=0.2.1
+aldryn-apphooks-config
+aldryn-boilerplates
+aldryn-common>=0.1.3
+aldryn-reversion>=0.0.2
+aldryn-translation-tools
```

Comment:
this changes are not needed, they should be installed during python setup.py install for aldryn_events not via test_requirements/base.txt